dev/install-skinny.sh

```diff
@@ -21,5 +19,5 @@ git fetch origin "$REF"
 git config advice.detachedHead false
 git checkout FETCH_HEAD
 OPTIONS=$(if pip freeze | grep -q "mlflow-skinny @"; then echo "--force-reinstall --no-deps"; fi)
-pip install --no-build-isolation $OPTIONS ./libs/skinny
```

Comment:
`--no-build-isolation` leads to a failure when the current environment doesn't satisfy the build system requirements (e.g. setuptools version is old).